### PR TITLE
fix: Update RBAC rules

### PR DESCRIPTION
**Pull request by wstrange**
Update the rbac to include events and volumesnapshots
Set the default docker repo to forgeops-public

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .PHONY: build install run test uninstall deploy manifest fmt vet generate docker-build release
 # Image URL to use all building/pushing image targets
-DEFAULT_IMG = gcr.io/engineering-devops/ds-operator
+DEFAULT_IMG = gcr.io/forgeops-public/ds-operator
```
Comment (jrcast) [inline Makefile:3]: I assume this will be accompanied by a cloudbuild change.

**Review by jrcast — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (maxres-fr) [reply Makefile:3]: This is already done

**Pull request merged**